plugins/amd-gpu/fu-amd-gpu-device.c

```diff
@@ -132,12 +133,8 @@ fu_amd_gpu_device_setup(FuDevice *device, GError **error)
 	fu_device_set_version_raw(device, vbios_info.version);
 	ver = fu_strsafe((const gchar *)vbios_info.vbios_ver_str, sizeof(vbios_info.vbios_ver_str));
 	fu_device_set_version(device, ver);
-
-	if (!fu_device_has_flag(device, FWUPD_DEVICE_FLAG_INTERNAL)) {
-		g_autofree gchar *model = NULL;
-		model = fu_strsafe((const gchar *)vbios_info.name, sizeof(vbios_info.name));
-		fu_device_set_name(device, model);
-	}
+	model = fu_strsafe((const gchar *)vbios_info.name, sizeof(vbios_info.name));
+	fu_device_set_description(device, model);
```

Comment:
summary surely? description is supposed to be in AppStream markup and be multiline paragraphs.